### DISCUSSION TITLE
How to deal with thiswarning

### DISCUSSION THREAD

**Discussion by liujing007008** [Chia-Network/chia-blockchain#6999] [Support]
![74CB3554-AD74-4A4E-8CA1-A4EFB1F7509F](https://user-images.githubusercontent.com/51757841/123545181-60d51200-d789-11eb-9962-781224e2502a.jpeg)
![44AD4100-96D3-4026-AC61-B9DDB4C236D5](https://user-images.githubusercontent.com/51757841/123545183-66caf300-d789-11eb-93c5-25a29635168e.jpeg)
How can I do with this warning?

**Comment by loppefaaret** [comment]
you have to look into what could possible cause it, do you have a potential bottleneck somewhere along the IO path ?

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
